docs/05-technical-reference/00-custom-resources/svls-01-function.md

```diff
@@ -100,25 +112,40 @@ This table lists all the possible parameters of a given resource together with t
-| **spec.minReplicas**                     |       No       | Defines the minimum number of Function's Pods to run at a time.  |
-| **spec.maxReplicas**                     |       No       | Defines the maximum number of Function's Pods to run at a time.    |
-| **spec.resources.limits.cpu**            |       No       | Defines the maximum number of CPUs available for the Function's Pod to use.      |
-| **spec.resources.limits.memory**         |       No       | Defines the maximum amount of memory available for the Function's Pod to use.      |
-| **spec.resources.requests.cpu**          |       No       | Specifies the number of CPUs requested by the Function's Pod to operate.       |
-| **spec.resources.requests.memory**       |       No       | Specifies the amount of memory requested by the Function's Pod to operate.               |
-| **spec.buildResources.limits.cpu**            |       No       | Defines the maximum number of CPUs available to use for the Kubernetes Job's Pod responsible for building the Function's image.      |
-| **spec.buildResources.limits.memory**         |       No       | Defines the maximum amount of memory available for the Job's Pod to use.      |
-| **spec.buildResources.requests.cpu**          |       No       | Specifies the number of CPUs requested by the build Job's Pod to operate.       |
-| **spec.buildResources.requests.memory**       |       No       | Specifies the amount of memory requested by the build Job's Pod to operate.               |
+| **metadata.labels**                          |       No       | Specifies the Function's Pod labels.    |
 | **spec.runtime**                         |       No       | Specifies the runtime of the Function. The available values are `nodejs14`, `nodejs16`, and `python39`. It is set to `nodejs14` unless specified otherwise.  |
 | **spec.runtimeImageOverride**                 |       No       | Specifies the runtimes image which must be used instead of the default one. |
-| **spec.type**                          |      No       | Defines that you use a Git repository as the source of Function's code and dependencies. It must be set to `git`. |
-| **spec.source**                          |      Yes       | Provides the Function's full source code or the name of the Git directory in which the code and dependencies are stored.     |
-| **spec.baseDir**                          |      No       | Specifies the relative path to the Git directory that contains the source code from which the Function will be built​. |
-| **spec.reference**                        |      No       | Specifies either the branch name or the commit revision from which the Function Controller automatically fetches the changes in Function's code and dependencies. |
+| **spec.source**                               |      Yes       | Defines Function's source code or Git repository with it. |
```

Comment:
I would describe it:
Contains specification of function. Only one configuration is allowed

Reply:
IMO whole CRD "Contains source specification of function" :)
I have no idea for good description of this aggregate field.

Reply:
```
Contains the function specification. Only one specification is allowed.
```

Reply:
done